app/poro/colfax_tour.rb

```diff
@@ -11,5 +11,6 @@ def colfax_coordinates
     latitudes = StreetArt.where('address ILIKE ?', '%Colfax%').pluck(:latitude)
     longitudes = StreetArt.where('address ILIKE ?', '%Colfax%').pluck(:longitude)
     coordinates = latitudes.zip(longitudes)
+    arrange_coordinates_by_latitude(coordinates)
```

Comment:
I chose whether to use `by_latitude` or `by_longitude` for the tours based on the orientation of the main drag where art is found.